aerialist/px4/drone_test.py

```diff
@@ -148,6 +151,7 @@ def plot(
                 obstacles=None
                 if test.simulation is None
                 else test.simulation.obstacles,
+                upload_dir=test.agent.path,
                 filename=filename,
             )
```

Comment:
Added the upload_dir as another param to the method to upload the plot into the modified test.agent.path folder. Since the Trajectory class doesn't have Dronetest as an attribute, I passed this as another parameter.

Reply:
can't we use WEBDAV_DIR for it?